pkg/options/options.go

```diff
@@ -63,6 +63,11 @@ const (
 	// - Attach
 	// It indicates which IO path to use to complete user IO
 	OptionsFastpath = "FASTPATH_STATE"
+	// OptionsForceUnmount is an option to the following Opentstorage Volume API
+	// - Unmount
+	// It is used to issue an umount system call to the requested path even
+	// if the entry for the path is not present in the mount table
+	OptionsForceUnmount = "FORCE_UNMOUNT"
```

Comment:
There is an actual force flag that is passed to the Unmount function as part of the "flags" argument. So this name is a bit confusing. Instead can we rename this to OptionsUnmountOnErrNoEnt and be more explicit of its usage